src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/sourceSuggestionExamples.tsx

```diff
@@ -11,76 +11,43 @@ type Props = {
   sourceName: string;
 };
 
-type State = {
-  isOpen: boolean;
-};
-
-class SourceSuggestionExamples extends React.Component<Props, State> {
-  state: State = {isOpen: false};
-
-  toggleModal = () => {
-    this.setState({isOpen: !this.state.isOpen});
-  };
-
-  stopPropagation = (e: React.MouseEvent<HTMLSpanElement>) => {
-    // Necessary to stop propagation of click events from modal that we can't
-    // catch otherwise.
-    e.stopPropagation();
-  };
-
-  render() {
-    const {isOpen} = this.state;
-    const {examples, sourceName} = this.props;
-
-    return (
-      <Wrapper onClick={this.stopPropagation}>
-        <Button size="xsmall" onClick={this.toggleModal}>
-          {t('examples')}
-        </Button>
-        {isOpen && (
-          <StyledModal show onHide={this.toggleModal}>
-            <Modal.Header closeButton>
-              {t('Examples for %s in current event', <code>{sourceName}</code>)}
-            </Modal.Header>
-            <Modal.Body>
-              {examples.map(example => (
-                <pre key={example}>{example}</pre>
-              ))}
-            </Modal.Body>
-          </StyledModal>
-        )}
-      </Wrapper>
-    );
-  }
-}
+const SourceSuggestionExamples = ({examples, sourceName}: Props) => (
+  <Wrapper>
+    <ExampleCard
+      position="right"
+      header={t('Examples for %s in current event', <code>{sourceName}</code>)}
+      body={examples.map(example => (
+        <pre key={example}>{example}</pre>
+      ))}
+    >
+      <Content>
+        {t('See Example')} <IconQuestion size="xs" />
+      </Content>
+    </ExampleCard>
+  </Wrapper>
+);
 
 export default SourceSuggestionExamples;
 
-const StyledModal = styled(Modal)`
-  .modal-dialog {
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%) !important;
-    margin: 0;
-    z-index: 1003;
-    @media (max-width: ${p => p.theme.breakpoints[0]}) {
-      width: 100%;
-    }
-  }
+const ExampleCard = styled(Hovercard)`
+  width: 400px;
 
-  .modal-body {
-    max-height: 500px;
-    overflow-y: auto;
-    padding: ${space(3)} ${space(4)};
-    margin: -${space(3)} -${space(4)};
+  pre:last-child {
+    margin: 0;
   }
+`;
 
-  .close {
-    outline: none;
-  }
+const Content = styled('span')`
+  display: inline-grid;
+  grid-template-columns: repeat(2, max-content);
+  align-items: center;
+  gap: ${space(0.5)};
```

Comment:
```suggestion
  grid-gap: ${space(0.5)};
```

Reply:
gap is actually full supported, and we should probably switch to it.

But I am all for consistency, so good call.